server.js

```diff
@@ -0,0 +1,246 @@
+const fs = require('fs');
+const http = require('http');
+const querystring = require('querystring');
+const check = require('./check');
+const element = require('./buildelement');
+const index = require('./reIndex');
+const authorize = require('./authorize')
+
+const PORT = process.env.PORT || 8080;
+
+//creates and starts server
+const server = http.createServer(function (req, res) {
+  let path;
+  let body = '';
+  //reads directory and checks if uri is the same as a file without the filetype
+  fs.readdir('./public/', function (err, files) {
```

Comment:
This readdir isn't really needed here. it's faster to check for a single file after you've figure out what file you're looking for.